tests/login.test.js

```diff
@@ -0,0 +1,80 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import env from 'dotenv';
+import app from '../src/app'; // assuming your app's entry point is in app.js
+// eslint-disable-next-line import/named
+import db, { sequelize } from '../src/database/models';
+
+const { expect } = chai;
+// eslint-disable-next-line import/named
+
+env.config();
+sequelize.authenticate();
+
+chai.should();
+chai.use(chaiHttp);
+
+const testUserData = {
+  username: 'user',
+  email: `user${Math.random() * 1000000}@mail.com`,
+  password: '123@User123',
+};
+
+let verifyEmailToken = '';
+
+before('Clear everyting from test db before testing', async () => {
+  if (process.env.NODE_ENV === 'test') {
+    await db.users.destroy({
+      truncate: true,
+    });
+  }
```

Comment:
I think this test needs to go after
because you gonna need to test for the `admin` and `sellers`
or you can delete
```
import { Op } from 'sequelize';
.
.
.
where: { 
    role: { 
        [Op.not]: ['admin', 'seller'] 
    } 
}
```

Reply:
What do you mean test for the **admin** and **seller** ?
That line is for removing all the users in the **testing database* before testing the login. 
So if somebody needs to test for **admins** or **sellers** they should make sure they create them in the tests, 
for example to test login I create users using the signup route, and verify them using the verify route. And in general test should not depend on the data in database created by other tests, or people.

Reply:
you can't create an admin without admin,
so you don't need to delete all users because its gonna block others for testing.
example: in few days later some one will get the task to create a seller and we know that requires the admin account to create a seller,
so with that there will be no admin to create a seller, so you will block him/her